github/doc.go

```diff
@@ -71,11 +71,8 @@ limited to 60 requests per hour, while authenticated clients can make up to
 that are not issued on behalf of a user, use the
 UnauthenticatedRateLimitedTransport.
 
-The Rate method on a client returns the rate limit information based on the most
-recent API call. This is updated on every call, but may be out of date if it's
-been some time since the last API call and other clients have made subsequent
-requests since then. You can always call RateLimits() directly to get the most
-up-to-date rate limit data for the client.
+You can always call RateLimits() directly to get the most up-to-date
+rate limit data for the client.
```

Comment:
In addition to removing the outdated part about `Rate` method on github client, I think we should put in its place a mention of the best way to access the remaining rate limit, and that is by accessing the `resp.Rate` of the previous call.

That [used to be](https://github.com/google/go-github/commit/2a4b92075e5c5e42ee0e55819b70f49b02c1ac64#diff-9bd6c2575adb220521a096df92e2d502L390) already written as part of the removed `Rate` method:

> Use the Response.Rate returned from most recent API call.

So, how about something like this:

> The returned Response.Rate value contains the rate limit information from the most recent API call. You can use RateLimits to get the most up-to-date rate limit data for the client.